### PR TITLE
[WIP]Update of script http 011

### PR DESCRIPTION
**Pull request by istoimenova**

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

File: test_scripts/Polices/build_options/011_ATF_P_TC_PTS_Define_URL_to_send_PTS_HTTP.lua [hunk 1]
```diff
@@ -94,26 +94,38 @@ end
 commonFunctions:newTestCasesGroup("Preconditions")
 
 function Test:ValidatePTS()
-  if ptu.policy_table.consumer_friendly_messages.messages then
-    self:FailTestCase("Expected absence of 'consumer_friendly_messages.messages' section in PTS")
+  if(ptu == nil) then
```
Comment (dboltovskyi) [inline test_scripts/Polices/build_options/011_ATF_P_TC_PTS_Define_URL_to_send_PTS_HTTP.lua]: @istoimenova, Currently there is an issue with ATF if 2 events with the same name are registered with Pin() option. Issue happens sometimes and leads to instability in scripts.
Please get rid of Pin() in lines 84 - 92

Comment (dboltovskyi) [inline test_scripts/Polices/build_options/011_ATF_P_TC_PTS_Define_URL_to_send_PTS_HTTP.lua]: @istoimenova, Please add additional condition on blank ptu

**Comment by istoimenova** [comment]
[WIP] - Clarified with Igor Kovalenko

File: test_scripts/Polices/build_options/011_ATF_P_TC_PTS_Define_URL_to_send_PTS_HTTP.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/Polices/build_options/011_ATF_P_TC_PTS_Define_URL_to_send_PTS_HTTP.lua]: @dboltovskyi, done in f544a6a425e72e5957c0ff81ce5b152895f5b0e5

Comment (istoimenova) [inline test_scripts/Polices/build_options/011_ATF_P_TC_PTS_Define_URL_to_send_PTS_HTTP.lua]: @dboltovskyi, done in f544a6a425e72e5957c0ff81ce5b152895f5b0e5

**Pull request merged**